contracts/state/State.sol

```diff
@@ -459,7 +469,16 @@ contract State is Ownable2StepUpgradeable, IState {
      * @param defaultIdType default id type
      */
     function _setDefaultIdType(bytes2 defaultIdType) internal {
+        _stateData.isIdTypeSupported[defaultIdType] = true;
         _defaultIdType = defaultIdType;
         _defaultIdTypeInitialized = true;
     }
+
+    /**
+     * @dev Set IdType  setter
+     * @param idType id type
+     */
+    function setIdType(bytes2 idType) public onlyOwner {
```

Comment:
Should be renamed to let's say `setIdTypeSupported` with additional boolean param `supported`, so that we could disable support for some idType.